apps/files/src/components/Collaborators/EditCollaborator.vue

```diff
@@ -21,7 +18,8 @@
           <oc-spinner :aria-label="$gettext('Saving Share')" class="uk-position-small uk-position-center-left" size="xsmall"/>
           <span :aria-hidden="true" class="uk-margin-small-left" v-translate>Saving Share</span>
         </oc-button>
-        <oc-button v-else variation="primary" :disabled="!$_hasChanges" :aria-label="$gettext('Save Share')" @click="$_ocCollaborators_saveChanges">
+        <oc-button v-else variation="primary" :disabled="!$_hasChanges" :aria-label="$gettext('Save Share')"
```

Comment:
I know this was missing there already before your PR but since you're touching this now, could you pls add also key to `v-if` and `v-else`?

Reply:
Done. :-) Same for the buttons in `NewCollaborator.vue`. Will keep an open eye about this in the future.

Reply:
Nice! Thanks!